tests/unit/test_lint.py

```diff
@@ -75,3 +77,48 @@ def test_nclcodestyle():
         """))
 
     assert success, "Your NCL code does not follow our formatting standards."
+
+
+def test_r_lint():
+    """Test R lint."""
+    environ = dict(os.environ)
+    try:
+        os.environ["LINTR_COMMENT_BOT"] = "FALSE"
+        package_root = os.path.dirname(
+            os.path.dirname(os.path.dirname(__file__))
+        )
+        checker = os.path.join(package_root, 'tests', 'unit', 'check_r_code.R')
+        if six.PY3:
```

Comment:
Why the different implementation for Python 3 and 2? From the subprocess documentation, it looks like both implementations work in both versions of Python. Maybe you can just use [`subprocess.check_output`](https://docs.python.org/3.7/library/subprocess.html#subprocess.check_output) instead, that only needs one line.

Reply:
`subprocess.run` is not available in python 2.7 and it is the recommended way in python 3.5+.

Checkotput will do the trick